mmdt/polls/admin.py

```diff
@@ -7,6 +9,22 @@ class ChoiceInline(admin.TabularInline):
     model = Choice
     extra = 3
 
+def export_to_csv(modeladmin, request, queryset):
```

Comment:
Please write the function doc in new function.

 for example.
```
def export_to_csv(modeladmin, request, queryset):
    """
    function def
    :param modeladmin: 
    :param request: 
    :param queryset: 
    :return: 
    """
```

Reply:
I am not clear what this means ama. Am I supposed to write documentation for this function like this?
```
def export_to_csv(modeladmin, request, queryset):
    """
    export_to_csv
    :param modeladmin: The ModelAdmin instance.
    :param request: The current request.
    :param queryset: The queryset containing selected poll results.
    :return: A CSV file response containing the poll results.
    """

Reply:
yes ! This code base is written by students and didn't put any functions docs, So I am thinking we better start to write the function docs this time.
```
def export_to_csv(modeladmin, request, queryset):
    """
    export the polls data to csv file.
    :param modeladmin: The ModelAdmin instance.
    :param request: The current request.
    :param queryset: The queryset containing selected poll results.
    :return: A CSV file response containing the poll results.
    """
```